recipes/hermes/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "2024-04-29":
```

Comment:
For Conan not to get confused, we'll need to remap this to `2024.04.29`, else it will think that this is version `2024` with prerelease version `04` which which has a prerelese `29` :)